### PR TITLE
Minor fix & preparing for sample data fixtures

### PR DESCRIPTION
**Pull request by danielgaldev**
Issue: DevTeamSCH/vikoverflow#8

Changes:
* Fix `__str__` in Profile
* Add [natural key](https://docs.djangoproject.com/en/2.1/topics/serialization/#natural-keys) to Profile (easier to write fixtures)

### CODE REVIEW

File: src/load_sample_data.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+from django.contrib.auth.models import User
+
+FIXTURES = [
+    'users',
+    'profiles',
+    'questions'
+]
+
+for fixture in FIXTURES:
+    call_command('loaddata', 'src/vikoverflow/fixtures/' + fixture)
```
Comment (barni2000) [inline src/load_sample_data.py]: use os.path.join instead of +

**Comment by danielgaldev** [comment]
Please elaborate. Do you mean tests?

**Comment by barni2000** [comment]
No i mean you can use list, for and models for load initial data.

**Comment by barni2000** [comment]
```python
from . import models

asd = models.Asd(asd="Asd")

asd.save()
# ETC.
```

**Comment by danielgaldev** [comment]
Redesigning idea, closing until then.

**Pull request closed without merging**